fullscreen/model/remove-first-manual.html

```diff
@@ -23,6 +23,9 @@
             assert_equals(document.fullscreenElement, last);
             assert_equals(event.target, last);
             first.remove();
+            // Both /first/ and /last/ were removed from the top layer and
+            // thus the fullscreen element synchronously becomes null.
+            assert_equals(document.fullscreenElement, null);
```

Comment:
By doing that synchronously, I mean we reset the document state to non-fullscreen immediately (`fullscreenElement` becomes null, and `:fullscreen` matches nothing) without waiting for the window to change.

I'm fine with the spec way, but as I mentioned before, since I don't think this is important, I'm not going to change that anytime soonish... If someone submits patch for that, we may accept. But I myself don't have much time working on this recently.